#! /bin/sh -e
## 16_add_maatuska.dpatch
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Add maatuska as eighth v3 directory authority

if [ $# -lt 1 ]; then
    echo "`basename $0`: script expects -patch|-unpatch as argument" >&2
    exit 1
fi

[ -f debian/patches/00patch-opts ] && . debian/patches/00patch-opts
patch_opts="${patch_opts:--f --no-backup-if-mismatch} ${2:+-d $2}"

case "$1" in
    -patch) patch -p1 ${patch_opts} < $0;;
    -unpatch) patch -R -p1 ${patch_opts} < $0;;
    *)
        echo "`basename $0`: script expects -patch|-unpatch as argument" >&2
        exit 1;;
esac

exit 0
#From 6c49b6bb7735335230ade722d1eccdadd652e938 Mon Sep 17 00:00:00 2001
#From: Karsten Loesing <karsten.loesing@gmx.net>
#Date: Thu, 10 Jun 2010 16:56:08 +0200
#Subject: [PATCH] Add maatuska as eighth v3 directory authority.
#
#---
# changes/maatuska-new-v3auth |    3 +++
# src/or/config.c             |    3 +++
# 2 files changed, 6 insertions(+), 0 deletions(-)
# create mode 100644 changes/maatuska-new-v3auth
#
#diff --git a/changes/maatuska-new-v3auth b/changes/maatuska-new-v3auth
#new file mode 100644
#index 0000000..9508b30
#--- /dev/null
#+++ b/changes/maatuska-new-v3auth
#@@ -0,0 +1,3 @@
#+  o New directory authorities:
#+    - Set up maatuska (run by Linus Nordberg) as the eighth v3 directory
#+      authority.
@DPATCH@
diff --git a/src/or/config.c b/src/or/config.c
index 26e42e7..7966557 100644
--- a/src/or/config.c
+++ b/src/or/config.c
@@ -922,6 +922,9 @@ add_default_trusted_dir_authorities(authority_type_t type)
       "193.23.244.244:80 7BE6 83E6 5D48 1413 21C5 ED92 F075 C553 64AC 7123",
     "urras orport=80 no-v2 v3ident=80550987E1D626E3EBA5E5E75A458DE0626D088C "
       "208.83.223.34:443 0AD3 FA88 4D18 F89E EA2D 89C0 1937 9E0E 7FD9 4417",
+    "maatuska orport=80 no-v2 "
+      "v3ident=49015F787433103580E3B66A1707A00E60F2D15B "
+      "213.115.239.118:443 BD6A 8292 55CB 08E6 6FBE 7D37 4836 3586 E46B 3810",
     NULL
   };
   for (i=0; dirservers[i]; i++) {
-- 
1.7.1

